### DISCUSSION TITLE
LU2-10A

### DISCUSSION THREAD

**Discussion by MrJanne** [arkypita/LaserGRBL#1990] [Q&A]
I have an Aufero Laser 1, and was thinking of changing the Laser Module to an LU2-10A. But it is not included in **LaserGRBL** as an option. How do I make it work? What settings are needed?

**Comment by Tribo-2dot0** [comment]
A tabela de máquinas incluída na lista Material DB é apenas uma referência de partida de parâmetros para gravações e cortes. A máquina fucionará normalmente e os parâmetros você pode usar de algum outro que seja semelhante e, aos poucos, ir montando seus próprios parâmentros incluindo-os nas ultimas linhas da tabela.

**Reply by MrJanne** [reply]
Ok, vou tentar algo parecido. Obrigado pela ajuda até agora.
